.github/workflows/asan.yaml

```diff
@@ -26,10 +31,14 @@ jobs:
             {
               "build": {
                 "mixin": ["asan-gcc"],
-                "cmake-args": ["-DCMAKE_BUILD_TYPE=Debug"]
+                "cmake-args": [
+                   "-DCMAKE_BUILD_TYPE=Debug"]
               }
             }
           colcon-mixin-repository: https://raw.githubusercontent.com/colcon/colcon-mixin-repository/master/index.yaml
+      - name: debug_ws
```

Comment:
What is this for?

Reply:
My bad this was for me to debug the workflow itself. Forgot to remove it.